en/VALUES.md

```diff
@@ -1,20 +1,20 @@
 ### FolksWhoCode Community Values
 
-The FolksWhoCode organization aims to promote an inclusive and diverse space 
+The FolksWhoCode organization aims to promote an inclusive and diverse space
 within the projects of the organization and wherever else our admins and users
 have a voice as advocates that can make an impact.
 
 Our projects are beginner friendly by default, and it is our job to make new
-contributors feel welcome when arriving to our repos and contributing. Our 
+contributors feel welcome when arriving to our repos and contributing. Our
 contribution guidelines reflect this in detail.
 
 Admin/senior/privileged users will act as mentors/sponsors of newcomers and will
-make sure their experience is satisfactory and enriching throughout their 
+make sure their experience is satisfactory and enriching throughout their
 interactions with our organization.
 
 When reviewing contributions, we will actively fight unconscious bias with any
 resources we have at hand to make the process as fair as we can.
 
-Our community and all of its repos aim to be a safe space, and our code of 
-conduct reflects that violence, harrassment or hate speech will not be 
-tolerated.
\ No newline at end of file
+Our community and all of its repos aim to be a safe space, and our code of
+conduct(https://github.com/folkswhocode/base/blob/master/en/CODE_OF_CONDUCT.md)
```

Comment:
This will not be a link if you don't put the link text inside square brackets (`[code of conduct](https://...)`) :)